internal/pkg/xds/adss/grpc_server.go

```diff
@@ -57,29 +49,28 @@ func NewServer(opts *ServerOpts, pushRequests <-chan xds.PushRequest, onNewSubsc
 		grpc:         grpcServer,
 		ads:          ads,
 		pushRequests: pushRequests,
-		opts:         opts,
 	}
 }
 
 // Run starts the gRPC server and the controllers.
 func (s *Server) Run(ctx context.Context) error {
 	var routinesGroup errgroup.Group
 
-	listener, err := net.Listen("tcp", fmt.Sprintf(":%d", s.opts.Port))
+	listener, err := net.Listen("tcp", ":15080")
```

Comment:
There may not currently be a need to customise this port number, but maybe `port` could be added to the `Server` type and its constructor function?

Reply:
I intentionally hardcoded the container port, because users can customize Service port. But in the vast majority of cases, users will customize only the Gateway port.